src/pkgmanagers/pnpm.js

```diff
@@ -0,0 +1,47 @@
+const unparse = require('../unparse')
+const skipArgs = require('../skipArgs')
+
+function getCommand(action) {
+  // Derived from pnpm-audit-report
+  // TODO: share the code
+  if (action.action === 'install') {
+    const isDev = action.resolves[0].dev
+    return `pnpm install ${isDev ? '--save-dev ' : ''}${action.module}@${action.target}`
+  } else {
+    return `pnpm update ${action.module} --depth ${action.depth}`
+  }
+}
+
+module.exports = {
```

Comment:
Doesn't look like it's any different from npm. Is this just 'p' added to every 'npm' in the code, or I missed the changes?

Is that enough for it to work?

Reply:
@naugtur PNPM commands are almost similar to NPM. It works. We are using my fork in production.